plugins/providers/shield/client.go

```diff
@@ -233,7 +196,7 @@ func (c *client) GetOrganizations(ctx context.Context) ([]*Organization, error)
 	return organizations, err
 }
 
-func (c *client) GrantTeamAccess(ctx context.Context, resource *Team, userId string, role string) error {
+func (c *client) GrantTeamAccess(ctx context.Context, resource *Group, userId string, role string) error {
```

Comment:
Same as above, function name should be GrantGroupAccess

Reply:
ok